src/components/panel/assets/panel.scss

```diff
@@ -0,0 +1,37 @@
+@import '~theme/base/variables';
+
+.modal {
```

Comment:
It seems like basic styles for the panel (without rendering it in a modal) are missing.

I'm sorry for not being clear in the origin issue, but when the `modal` prop is equals to `false`, the panel should be able to render in place, like in a sidebar or anywhere else.

Reply:
By only panel, do you mean just with backdrop and not positioned fixed?

Reply:
I think so

Reply:
@crysfel, @tureey I have added styles for non modal popup

Reply:
yeap!